grid/chat/channels/profile.supr

```diff
@@ -0,0 +1,191 @@
+name = "chat/channels"
+version = "1.0.0"
+
+"""
+Get Channels
+Retrieve paginated list of channels, e.g. text channels in slack or discord
+"""
+usecase GetChannels {
+  input {
+    """
+    Server
+    Representation of a server. Can be guild or server, depending on the provider capability.
+    """
+    server string
+
+    """
+    Types
+    List of categories of channels
```

Comment:
Does it list all channels if I omit this?

I am also somewhat concerned about a broad meaning of "type"; perhaps I might want to filter text vs. voice channels on Discord or something else. Consider renaming this to e.g. `visibility` or `privacy`

Documentation improvement:

```suggestion
    Channel Types Filter
    Limit the listing to either public or private channels; all channels are listed by default.
```

Reply:
Good point. In case of slack, it does list all channels if omitted, but it list just relevant channels in case of discord. Discord does not have any filtering parameters when listing guild channels, so we can only filter channel types after we get them.

Reply:
We can either extend this to contain relevant filtering types such as `text` or `voice` for discord and handle them in map, or create another filtering property for discord types. 🤷🏻‍♂️